ios/brave-ios/Sources/BraveWallet/Crypto/Transaction Confirmations/SaferSignTransactionContainerView.swift

```diff
@@ -48,7 +48,7 @@ struct SaferSignTransactionContainerView: View {
   ) {
     self.originInfo = parsedTransaction.transaction.originInfo
     self.network = parsedTransaction.network
-    self.fromAddress = parsedTransaction.fromAddress
+    self.fromAddress = parsedTransaction.fromAccountId.address
```

Comment:
Let's not depend on accountId.address as address will not be a part of AccountId "soon". Address will an optional part of AccountInfo struct.